core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4292,8 +4289,9 @@ definitions:
         description: The run cost of this workflow
   WorkflowStatus:
     properties:
-      done:
-        type: boolean
+      type: string
+      description: workflow status
+      enum: ["Queued", "Launching", "Submitted", "Running", "Failed", "Succeeded", "Aborting", "Aborted", "Unknown"]
```

Comment:
this is invalid; remove `properties`:

```
  WorkflowStatus:
    type: string
    description: workflow status
    enum: ["Queued", "Launching", "Submitted", "Running", "Failed", "Succeeded", "Aborting", "Aborted", "Unknown"]
```

try http://editor.swagger.io/ for validation! (yes, there are other validation errors in this yaml)